test/fw/bin/pbs_benchpress

```diff
@@ -340,6 +359,42 @@ if __name__ == '__main__':
             'Testsuites names require (see -t) along with -i option!')
         sys.exit(1)
 
+    try:
+        tc_failure_threshold = int(tc_failure_threshold)
+        if tc_failure_threshold < 0:
+            raise ValueError
+    except:
+        _msg = 'Invalid value provided for testcase failure threshold, '
+        _msg += 'please provide integer'
+        logging.error(_msg)
+        sys.exit(1)
+
+    try:
+        cumulative_tc_failure_threshold = int(cumulative_tc_failure_threshold)
+        if cumulative_tc_failure_threshold < 0:
+            raise ValueError
+    except:
+        _msg = 'Invalid value provided for cumulative-tc-failure-threshold, '
+        _msg += 'please provide integer'
+        logging.error(_msg)
+        sys.exit(1)
+
+    if cumulative_tc_failure_threshold < tc_failure_threshold:
+        _msg = 'Value for cumulative-tc-failure-threshould should'
```

Comment:
@arungrover Sorry for delay. I have addressed this comment. Please have look again.